ldap/servers/slapd/daemon.c

```diff
@@ -918,6 +920,41 @@ slapd_sockets_ports_free(daemon_ports_t *ports_info)
 #endif
 }
 
+/*
+ * slapi_eq_repeat_rel callback that checks that idletimeout has not expired.
+ */
+void
```

Comment:
I like this approach but I wonder why the previous approach is no longer working.
In slapd_deamon we are looping to call setup_pr_read_pds with a timeout (1s) on poll. 
Is idletime not longer tested because this function is no longer used ?

Reply:
setup_pr_read_pds is still called (by ct_list_thread) at least every seconds but it does not test for idle timeout !
  If you grep c_idletimeout in previous code, you see that the only place it is tested is in  handle_pr_read_ready 
    and ct_list_thread does not call handle_pr_read_ready if the poll timed out